packages/v-analyzer/package.yaml

```
@@ -0,0 +1,32 @@
+---
+name: v-analyzer
+description: V language server.
+homepage: https://github.com/v-analyzer/v-analyzer
+licenses:
+  - MIT
+languages:
+  - V
+categories:
+  - LSP
+
+source:
+  id: pkg:github/v-analyzer/v-analyzer@latest
+  asset:
+    - target: darwin_arm64
+      file: v-analyzer-darwin-arm64.zip
+      bin: v-analyzer
+    - target: darwin_x64
+      file: v-analyzer-darwin-x86_64.zip
+      bin: v-analyzer
+    - target: linux_x64
```

I believe it's built for GNU systems:
```suggestion
    - target: linux_x64_gnu
```